### ISSUE TITLE
'SebastianBergmann\CodeCoverage\TestCase' not found 

### ISSUE THREAD

**Issue by m1357l** [sebastianbergmann/php-code-coverage#454]
I installed version 4 of php-code-coverage, with the latest version of phpunit and php 5.6, but I am getting the following error when running a simple test:

Fatal error: Class 'SebastianBergmann\CodeCoverage\TestCase' not found in ... BuilderTest.php on line 17

**here is my composer.json**:
```
{
    "require" : {
        "php" : "5.6.*"
    },
    "require-dev": {
        "phpunit/phpunit": "5.4.6",
        "phpunit/php-code-coverage": "^4"
    }
}
```

**here are the first few lines of BuilderTest.php:**

```
namespace SebastianBergmann\CodeCoverage\Report;

use SebastianBergmann\CodeCoverage\TestCase;
use SebastianBergmann\CodeCoverage\Node\Builder;

class BuilderTest extends TestCase
```

Any ideas what the issue could be?

**Comment by sebastianbergmann** [comment]
Looks like your running this component's own test suite without its configuration.

**Comment by m1357l** [comment]
No terribly helpful, but I think the problem is because the dep is called 

    'SebastianBergmann\\CodeCoverage\\CodeCoverage' => $vendorDir . '/phpunit/php-code-coverage/src/CodeCoverage.php',

whereas the PHPStorm is looking for

'PHP_CodeCoverage_Filter' not found in /private/var/folders/2s/2n9d7269029_6nfn_df18bkm0000gp/T/ide-phpunit.php on line 320

If you can point me to a version/fork that contains PHP_CodeCoverage that would help I think.

**Comment by GM-Alex** [comment]
@m1357l I ran into the same issue and the solution is quite simple. You defined the directory containing the vendor dir as your test dir.

**Comment by OndraM** [comment]
@GM-Alex That was it, thanks! However, this is kind of confusing error.

**Comment by basilmusa** [comment]
Just exclude the "vendor" directory in your phpunit.xml file (incase you dont want to move it outside the folder).

**Comment by jxzhangxing** [comment]
> @m1357l I ran into the same issue and the solution is quite simple. You defined the directory containing the vendor dir as your test dir.

Sorry I'm a newbie of phpunit and I ran into this issue in my cakephp project. 
I don't quite understand how to deal with it. 
Could you explain it more explicitly?
Where should I put my test file and how should I fix the phpunit configuration?

**Comment by piRGoif** [comment]
Hello,

We have the same issue on our application...
Starting point was we figure that tests cannot be ran anymore in PHPStorm (PHPUnit was updated recently).
By launching the same command in a terminal, we got the very same exception as this issue.

We have a test/phpunit.dist.xml file, and vendor dir is located also under /test (see [on our GitHub repo](https://github.com/Combodo/iTop/tree/support/2.7/test)).
Excluding the vendor dir from within the phpunit.dist.xml file doesn't help : still the same error.

When launching tests, PHPStorm specify the directory. Using the same command but without the directory works. That is how our CI platform launches tests, at least there are no impact there.
In PHPStorm we are using the "Defined in the configuration file" [option](https://www.jetbrains.com/help/phpstorm/2022.1/run-debug-configuration-phpunit.html#testRunner).

I found [this comment](https://github.com/sebastianbergmann/phpunit/issues/2254#issuecomment-235659417) that seems to point exactly to the same problem, but I don't see any solution or workaround :(

Any information would be much welcome !
